libnopegl/src/atlas.c

```diff
@@ -169,10 +169,17 @@ struct texture *ngli_atlas_get_texture(const struct atlas *s)
     return s->texture;
 }
 
-void ngli_atlas_get_dimensions(const struct atlas *s, int32_t *dim)
```

Comment:
Typo `altas` in commit desc.

Reply:
`for an appropriate sampling of the bitmap.`  -> `to sample the bitmap correctly.`

Reply:
fixed